web-server/v0.4/src/pages/RunComparison/index.js

```diff
@@ -101,12 +101,13 @@ class RunComparison extends React.Component {
       exportModalVisible: false,
       defaultComponents: ['details', 'summary', 'table', 'timeseries'],
       selectedComponents: [],
-      pdfHeader: '',
-      pdfName: '',
+      pdfHeader: ' ',
```

Comment:
A few unrelated changes seem to have leaked into this PR - can we clean this up?